libsolutil/StringUtils.h

```diff
@@ -101,109 +101,111 @@ std::string joinHumanReadablePrefixed
 		return _separator + joinHumanReadable(_list, _separator, _lastSeparator);
 }
 
-/// Same as @ref formatNumberReadable but only for unsigned numbers
+/// Formats large numbers to be easily readable by humans.
+/// Returns decimal representation for smaller numbers; hex for large numbers.
+/// "Special" numbers, powers-of-two and powers-of-two minus 1, are returned in
+/// formulaic form like 0x01 * 2**24 - 1.
+/// @a T can be any integer value, will typically be u160, u256 or bigint.
```

Comment:
I don't want to be pedantic but it looks to me that `T` refers to a type like `u160` or `bigint`. It is the function param variable   `_value` that stores the integer value of type `T`.
```suggestion
/// @a T can be any integer type, will typically be u160, u256 or bigint.
```

Reply:
Sorry, I accidentally might have forgot to add this edit to my code. My bad. It's fixed now.

Reply:
No problem :)